src/main/java/com/xjeffrose/chicago/ChicagoObjectEncoder.java

```diff
@@ -76,10 +76,8 @@ protected void encode(ChannelHandlerContext ctx, Object msg, List<Object> out) t
     if (msg instanceof ChicagoMessage) {
       out.add(encode(ctx, (ChicagoMessage) msg));
     } else if (msg instanceof ByteBuf) {
-      //ByteBuf bb = ctx.alloc().directBuffer();
-      //bb.writeBytes(((ByteBuf)msg).readBytes(((ByteBuf) msg).readableBytes()));
-      //out.add(bb);
-      out.add(((ByteBuf) msg).retain());
```

Comment:
Yes, i commented it out as it was complaining at me during a poorly written test. I neglected to re-add once the test was fixed.